src/main/resources/application.properties

```diff
@@ -1,6 +1,9 @@
 # Actuator health check config
+management.endpoints.enabled-by-default=false
 management.endpoints.web.base-path=/
```

Comment:
the actuator basepath should be the same as the server.servlet.context-path, to keep consistent with other recent upgrades